Source/WebCore/dom/Document.cpp

```diff
@@ -6841,8 +6841,9 @@ void Document::initSecurityContext()
         return;
     }
 
-    contentSecurityPolicy()->copyStateFrom(ownerFrame->document()->contentSecurityPolicy());
-    contentSecurityPolicy()->updateSourceSelf(ownerFrame->document()->securityOrigin());
+    CheckedPtr contentSecurityPolicy = this->contentSecurityPolicy();
```

Comment:
Why isn't this just:
`CheckedPtr contentSecurityPolicy = checkedContentSecurityPolicy();` ?

Reply:
Why? I prefer my pattern. `checkedXXX()` functions are useful when there is no local variable but have no benefit when there is a local variable.